nautobot_floor_plan/filter_extensions.py

```diff
@@ -19,6 +20,17 @@ class RackFilterExtension(PluginFilterExtension):
         ),
     }
 
+class UniqueRackFilterExtension(PluginFilterExtension):
+    """Add a filter to ensure a Rack is only an option if it isn't installed."""
+
+    model = "dcim.rack"
+
+    filterset_fields = {
+        "nautobot_floor_plan_has_floor_plan_tile": RelatedMembershipBooleanFilter(
+            field_name="floor_plan_tile",
+            label="Floor plan",
+        ),
+    }
```

Comment:
Let's go ahead and add in the `filterform_fields` to add the filter to the UI as well.

Reply:
verified the filter is present in Rack filters.